src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    abstract void doWork();
```

Comment:
```suggestion
public abstract class Machine {
    abstract void doWork();
```
![image](https://github.com/mate-academy/jv-oop/assets/20326858/9179b223-868f-4d24-9f75-a38b3225b7e9)